skill-tree/src/test/java/com/RDS/skilltree/integration/SecurityContextIntegrationTest.java

```diff
@@ -27,6 +29,7 @@ public void testTokenIsNotPresent() {
     }
 
     @Test
+    @Disabled
```

Comment:
same here any reason...?

Reply:
Same reason here as well, disabling it since the endpoint currently is not protected as it uses dummy json approach